Sources/PrimerSDK/Classes/Data Models/PaymentResponse.swift

```diff
@@ -44,7 +44,7 @@ internal enum PaymentStatus: Int, Codable {
     }
 }
 
-internal enum RequiredActionName: String, Codable {
+public enum RequiredActionName: String, Codable {
```

Comment:
Because in the `ResultViewController` Manual + HUC  at the moment, we expose the whole `Payment.Response` object. Thus, the RequiredAction 😕 . Happy to review it later on.